src/components/PlayerView/events.ts

```diff
@@ -67,65 +67,281 @@ import {
  * Used to generate the specific events interface for each component.
  */
 interface EventProps {
+  /**
+   * Event emitted when an ad break has finished.
+   */
   onAdBreakFinished: AdBreakFinishedEvent;
+  /**
+   * Event emitted when an ad break has started.
+   */
   onAdBreakStarted: AdBreakStartedEvent;
+  /**
+   * Event emitted when an ad has been clicked.
+   */
   onAdClicked: AdClickedEvent;
+  /**
+   * Event emitted when an ad error has occurred.
+   */
   onAdError: AdErrorEvent;
+  /**
+   * Event emitted when an ad has finished.
+   */
   onAdFinished: AdFinishedEvent;
+  /**
+   * Event emitted when an ad manifest starts loading.
+   */
   onAdManifestLoad: AdManifestLoadEvent;
+  /**
+   * Event emitted when an ad manifest has been loaded.
+   */
   onAdManifestLoaded: AdManifestLoadedEvent;
+  /**
+   * Event emitted when an ad quartile has been reached.
+   */
   onAdQuartile: AdQuartileEvent;
+  /**
+   * Event emitted when an ad has been scheduled.
+   */
   onAdScheduled: AdScheduledEvent;
+  /**
+   * Event emitted when an ad has been skipped.
+   */
   onAdSkipped: AdSkippedEvent;
+  /**
+   * Event emitted when an ad has started.
+   */
   onAdStarted: AdStartedEvent;
+  /**
+   * Event emitted when casting to a cast-compatible device is available.
+   *
+   * @platform iOS, Android
+   */
   onCastAvailable: CastAvailableEvent;
+  /**
+   * Event emitted when the playback on a cast-compatible device was paused.
+   *
+   * @platform iOS, Android
+   */
   onCastPaused: CastPausedEvent;
+  /**
+   * Event emitted when the playback on a cast-compatible device has finished.
+   *
+   * @platform iOS, Android
+   */
   onCastPlaybackFinished: CastPlaybackFinishedEvent;
+  /**
+   * Event emitted when playback on a cast-compatible device has started.
+   *
+   * @platform iOS, Android
+   */
   onCastPlaying: CastPlayingEvent;
+  /**
+   * Event emitted when the cast app is launched successfully.
+   *
+   * @platform iOS, Android
+   */
   onCastStarted: CastStartedEvent;
+  /**
+   * Event emitted when casting is initiated, but the user still needs to choose which device should be used.
+   *
+   * @platform iOS, Android
+   */
   onCastStart: CastStartEvent;
+  /**
+   * Event emitted when casting to a cast-compatible device is stopped.
+   *
+   * @platform iOS, Android
+   */
   onCastStopped: CastStoppedEvent;
+  /**
+   * Event emitted when the time update from the currently used cast-compatible device is received.
+   *
+   * @platform iOS, Android
+   */
   onCastTimeUpdated: CastTimeUpdatedEvent;
+  /**
+   * Event emitted when a cast-compatible device has been chosen and the player is waiting for the device to get ready for
+   * playback.
+   *
+   * @platform iOS, Android
+   */
   onCastWaitingForDevice: CastWaitingForDeviceEvent;
+  /**
+   * Event emitted when the player is destroyed.
+   */
   onDestroy: DestroyEvent;
+  /**
+   * Called for each occurring player event.
+   */
```

Comment:
I would prefer a more consistent naming. The other functions are documented with 'event emitted', this one it's 'called' but functionality wise there is no major difference.

Reply:
Fixed in 736513c